### PR TITLE
Weather app

### PR DESCRIPTION
**Pull request by FelipeEnne**
Create a Weather app 

* Bundled with Webpack;
*  API to fetch weather data using Async JS;
* Use the DOM to display in the page;
* Linting with ESLINT.

Waiting for review :)

### CODE REVIEW

File: src/DOMmodel.js [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+  } else {
+    model.style.display = 'block';
+  }
+
+  const result = await data.json();
+
+  return result;
+}
+
+async function diplayData() {
```
Comment (bellom) [inline src/DOMmodel.js]: let practice good naming of a variable;
`displayData` looks great

File: src/DOMmodel.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  img.setAttribute('src', `http://openweathermap.org/img/wn/${data.weather[0].icon}@2x.png`);
+  temp.innerHTML = `${parseFloat(data.main.temp - 273).toFixed(1)} °C`;
+  feel.innerHTML = `Feels like ${parseFloat(data.main.feels_like - 273).toFixed(1)} °C`;
+  minmax.innerHTML = ` Min ${parseFloat(data.main.temp_min - 273).toFixed(1)} °C
+      - Max ${parseFloat(data.main.temp_max - 273).toFixed(1)} °C`;
+  humidity.innerHTML = `Humidity ${data.main.humidity} %`;
+  wind.innerHTML = `Wind ${data.wind.speed} meter/sec`;
+}
+
+export default diplayData;
```
Comment (bellom) [inline src/DOMmodel.js]: let practice good naming of variable;
`displayData` looks great

File: src/returnCity.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+function storeCity(city) {
+  const cit = JSON.stringify(city);
```
Comment (bellom) [inline src/returnCity.js]: let practice good naming of a variable;
`getCity` looks great right?

**Comment by FelipeEnne** [comment]
I had problems with Stickler and I used linter in local env.
I don´t know why the Stickler is not working.
But I run in the local and everything is fine, you can run the task npx-fix to see.

**Pull request merged**